doc/source/deploy-guide/configure-environment.rst

```diff
@@ -0,0 +1,150 @@
+Configure the deployment
+========================
+
+.. blockdiag::
+
+   blockdiag {
+
+     localhost [label="Prepare localhost"]
+     ses [label="Deploy SES\n(optional)"]
+     caasp [label="Deploy CaaSP\n(optional)"]
+     deployer [label="Deploy deployer\n(optional)"]
+     enroll_caasp [label="Enroll CaaSP\n(optional)"]
+     setup_caasp_workers [label="Setup CaaSP\nfor OpenStack"]
+     patch_upstream [label="Apply patches\nfrom upstream\n(for developers)"]
+     build_images [label="Build docker images\n(for developers)"]
+     deploy [label="Deploy Airship or\nOpenStack-Helm"]
+     configure_deployment [label="Configure deployment"]
+
+     localhost -> ses;
+
+     group {
+       color = "#EEEEEE"
+       label = "Setup hosts"
+       ses -> caasp;
+       caasp -> deployer [folded];
+       deployer -> enroll_caasp;
+     }
+     enroll_caasp -> configure_deployment [folded];
+     localhost -> configure_deployment[folded];
+
+     group {
+       color = "red"
+       configure_deployment
+     }
+
+     configure_deployment -> setup_caasp_workers;
+
+     group {
+       color = "#EEEEEE"
+       label = "Setup openstack/Setup airship"
+       setup_caasp_workers -> deploy, patch_upstream [folded];
+       patch_upstream -> build_images;
+       build_images -> deploy;
+     }
+   }
+
+All the files for the deployment are in a workspace, whose default location
+is `~/suse-osh-deploy`.
+
+This workspace is structured like an `ansible-runner` directory.
+
+It therefore contains:
+
+* an `inventory` folder
+* an `env` folder.
+
+Additionally, this folder will contain extra files necessary for the
+deployment.
+
+Configure the inventory
+-----------------------
+
+If you are bringing your own cluster, create an inventory based on our
+example, located in the `examples` folder.
+
+.. literalinclude:: ../../../examples/workdir/inventory/hosts.yml
+
+As you can see, this inventory only contains the group names.
+
+For each group, a `hosts:` key should be added, with, as value, each
+of the hosts you will need. For example:
+
+.. code-block:: yaml
+
+   caasp-admin:
+     hosts:
+       jevrard-57997-admin-x6sugiws4g34:
+         ansible_host: 10.86.1.144
```

Comment:
It might be better to paste here full `hosts.yml` as an example

Reply:
I do not believe so, it would encourage people to copy and paste blindly. I would rather have a simple example, and they build their own inventory.

Reply:
Ofc I can switch opinions, or do both.